pyproject.toml

```diff
@@ -0,0 +1,93 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "packitos"
+authors = [
+    { name = "Red Hat", email = "user-cont-team@redhat.com" },
+]
+description = "A set of tools to integrate upstream open source projects into Fedora operating system."
+readme = "README.md"
+license = "MIT"
+license-files = { paths = ["LICENSE"] }
+requires-python = ">=3.9"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Software Development",
+    "Topic :: Utilities",
+]
+dynamic = ["version"]
+keywords = [
+    "git",
+    "packaging",
+    "fedora",
+    "rpm",
+    "dist-git",
+]
+dependencies = [
+    "GitPython",
+    "PyYAML",
+    "cccolutils",
+    "click",
+    "copr",
+    "lazy_object_proxy",
+    "marshmallow",
+    "marshmallow-enum",
+    "munch",
+    "ogr",
+    "packaging",
+    "python-gnupg",
+    "requests",
+    "requests-kerberos",
+    "setuptools", # Required for pkg_resources. Should move to importlib
+    "specfile",
+    "tabulate",
+    "bodhi-client >= 7.0.0",
+    "koji",
+    "rpkg",
+    "cachetools",
+    "python-fedora",
+]
+
+[project.urls]
+Homepage = "https://github.com/packit/packit"
+
+[project.optional-dependencies]
+testing = [
+    "pytest",
+    "flexmock",
+    "deepdiff",
+    "distro",
+    "setuptools", # Required for test_upstream.py/test_get_version_macro
+    "setuptools-scm", # Required for tests using python-ogr.spec
+    "setuptools-scm-git-archive", # Required for tests using python-ogr.spec
+]
+
+[project.scripts]
+packit = "packit.cli.packit_base:packit_base"
+
+[tool.setuptools]
+script-files = ["_packitpatch"]
```

Comment:
Since https://github.com/packit/packit/pull/1926 has been merged, this is no longer necessary.

Reply:
It is already removed. You are reviewing an outdated version here. Please check again

Reply:
It's still part of 194dfa93e6db4b5dcf99efa700e634e3ed2725f0. If you consider that commit outdated, remove it from the history please.

Reply:
Ok, rebased the history for this. These will be squash-merged though right? So it would be hard to revert/reference the commits later.

Reply:
Ooh, thankfully I kept a logical commit history :D